pavelib/utils/test/suites/js_suite.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, *args, **kwargs):
         self.opts = kwargs
 
         suite = args[0]
-        self.subsuites = self._default_subsuites if suite == 'all' else [JsTestSubSuite(*args, **kwargs)]
```

Comment:
@salman2013 I'm not super familiar with the Karma test suites, can you explain what you're doing here?

Reply:
@feanil I was trying to remove Karma tests that's why i commented and not removed but now i reverted the changes as we are keeping the karma as it is.

Reply:
Should this be uncommented now as well?

Reply:
Yes, i have uncommented it.